pkg/runtimes/docker/util.go

```diff
@@ -175,7 +175,7 @@ func GetDockerClient() (*client.Client, error) {
 	}
 
 	newClientOpts := flags.NewClientOptions()
-	newClientOpts.Common.LogLevel = log.GetLevel().String() // this is needed, as the following Initialize() call will set a new log level on the global logrus instance
+	newClientOpts.Common.LogLevel = l.Log().GetLevel().String() // this is needed, as the following Initialize() call will set a new log level on the global logrus instance
```

Comment:
I guess this could be dropped now, as k3d itself doesn't use the global logrus instance anymore.. but it's not that important and maybe it could even be useful to cascade the log level down to the docker client :thinking: